master/buildbot/www/rest.py

```diff
@@ -337,6 +337,8 @@ def writeError(msg, errcode=404, jsonrpccode=None):
             request.write(json.dumps(dict(error=msg)))
 
         with self.handleErrors(writeError):
+            yield self.master.www.assertUserAllowed(request, tuple(request.postpath), request.method, {})
```

Comment:
Any comment on this change causing new behavior with existing configs using `AnyEndpointMatcher`?